components/ui/AppMosaic.vue

```diff
@@ -21,6 +21,13 @@
             v-html="description"
           />
           <!-- estlint-enable -->
+          <AppLink
+            v-if="cta"
+            v-bind="cta"
+            class="app-mosaic__element-copy-link copy__link"
+          >
+            {{ cta.label }}
+          </AppLink>
```

Comment:
Nice to have: Use the `v-text` binding:

```suggestion
          <AppLink
            v-if="cta"
            v-bind="cta"
            class="app-mosaic__element-copy-link copy__link"
            v-text="cta.label"
          />
```